### DISCUSSION TITLE
Allow customisation of names of formats appearing when `format-links` is `true`

### DISCUSSION THREAD

**Discussion by petrbouchal** [quarto-dev/quarto-cli#7457] [Feature Requests]
### Description

It would be nice if the names of additional formats as they appear under the Other formats heading in the sidebar could be customised.

Currently, these are hardcoded in the Quarto source code and cannot be customised via the `language` YAML param. 

Incidentally, Typst appears as Typst, not PDF, which for a non-technical user might obscure the actual format.

**Comment by cscheid** [comment]
Agreed, tracking in #7459 .
